lib/motors/controller.hpp

```diff
@@ -5,6 +5,7 @@
 
 namespace hyped::motors {
 enum class ControllerStatus { kControllerTemperatureExceeded, kUnrecoverableWarning, kNominal };
+enum class ControllerStates { initialisation, calibration, motorReady, motorAccelerating, motorBraking, runComplete, emergencyBreak };
```

Comment:
If it's constant states, I'm pretty sure the naming convention is to prefix the states with the letter k (i.e. `kInitialisation`, `kCalibration`, etc.)